pkg/etcd/etcd.go

```diff
@@ -261,9 +261,35 @@ func nameFile(config *config.Control) string {
 	return filepath.Join(DBDir(config), "name")
 }
 
+// clearReset removes the reset file
+func (e *ETCD) clearReset() error {
+	if err := os.Remove(e.ResetFile()); err != nil && !os.IsNotExist(err) {
```

Comment:
```suggestion
    return os.Remove(e.ResetFile()); err != nil && !os.IsNotExist(err)
```

Reply:
I don't think that's valid syntax, is it? Is there a better pattern for filtering out a particular type of error?